src/txn/txn.c

```diff
@@ -1714,6 +1714,13 @@ __wt_txn_commit(WT_SESSION_IMPL *session, const char *cfg[])
      */
     if (!readonly)
         WT_IGNORE_RET(__wt_cache_eviction_check(session, false, false, NULL));
+
+    /*
+     * Stable timestamp cannot be moved concurrently to beyond and equal to the prepared
```

Comment:
How about something like the following?
`Stable timestamp cannot be concurrently increased beyond or equal to the prepared transaction's durable timestamp.`